lib/rubocop/cop/lint/overwrite_by_rescue.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Checks for overwriting an exception with an exception result by use `rescue =>`.
+      #
+      # @example
+      #
+      #   # bad
+      #   begin
+      #     something
+      #   rescue => StandardError
+      #   end
+      #
+      #   # good
+      #   begin
+      #     something
+      #   rescue StandardError
+      #   end
+      #
+      class OverwriteByRescue < Base
```

Comment:
You've used `Overwrite` in the class name and `Override` in the message. :)

Reply:
Oh. I had missed it💦 Thank you!